chain/sync.go

```diff
@@ -1336,12 +1336,12 @@ func (syncer *Syncer) collectChain(ctx context.Context, ts *types.TipSet) error
 	return nil
 }
 
-func VerifyElectionPoStVRF(ctx context.Context, evrf []byte, rand []byte, worker address.Address) error {
-	if err := gen.VerifyVRF(ctx, worker, rand, evrf); err != nil {
-		return xerrors.Errorf("failed to verify post_randomness vrf: %w", err)
+func VerifyElectionPoStVRF(ctx context.Context, worker address.Address, rand []byte, evrf []byte) error {
+	if build.InsecurePoStValidation {
+		return nil
```

Comment:
let's put a `log.Warn` in here, these sorts of conditions make me nervous

Reply:
The log will already be triggered on line 545 (in `ValidateBlock`). Would you rather it double-logged for extra-safety?